src/main/java/soot/toolkits/graph/ExceptionalUnitGraph.java

```diff
@@ -667,8 +670,6 @@ private void buildHeadsAndTails(Set<Unit> additionalHeads) {
     Collection<ExceptionDest> result = unitToExceptionDests.get(u);
     if (result == null) {
       ExceptionDest e = new ExceptionDest(null, null) {
-        private ThrowableSet throwables;
```

Comment:
ExceptionDest already defines this field, it was just private so it was not accessible. So I removed the private from that field and remove this field so there would be no duplication.